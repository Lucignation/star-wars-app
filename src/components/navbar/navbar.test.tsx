import Navbar from './navbar.component';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

const mockProps = {
  backBtn: false,
};

describe('Navbar', () => {
  it('renders a snapshot of <Navbar /> when passed into a component', () => {
    const navbar = renderer
      .create(
        <Router>
          <Navbar {...mockProps} />
        </Router>
      )
      .toJSON();

    expect(navbar).toMatchSnapshot();
  });
});
