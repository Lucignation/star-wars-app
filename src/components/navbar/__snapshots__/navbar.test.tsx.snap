// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders a snapshot of <Navbar /> when passed into a component 1`] = `
<nav
  className="navbar_container"
>
  <div
    className="navbar_right_items"
  >
    <div
      className="navbar_notification_avatar"
    >
      <div
        className="navbar_notification_icon"
      >
        <img
          alt="user notification"
          src="notification.svg"
        />
      </div>
      <div
        className="navbar_avatar_icon"
      >
        <img
          alt="account profile"
          src="account.svg"
        />
      </div>
      <p
        className="navbar_user_name"
      >
        John Doe
      </p>
    </div>
    <div
      className="navbar_action_icon"
    >
      <img
        alt="more actions"
        src="three_dotted.svg"
      />
    </div>
  </div>
</nav>
`;
